app/src/test/java/com/futurice/freesound/feature/audio/DefaultObservableExoPlayerTest.java

```diff
@@ -164,7 +164,8 @@ ArrangeBuilder withEndedExoPlayer() {
         }
 
         ArrangeBuilder withExoPlayerStateStreamEvent(boolean playWhenReady, int state) {
-            exoPlayerStateOnceAndStream.onNext(ExoPlayerState.create(playWhenReady, state));
+            exoPlayerStateOnceAndStream.onNext(
+                   new ExoPlayerState(playWhenReady, state));
```

Comment:
I get an error/warning here about "use of Kotlin internal declaration from different module".